### ISSUE TITLE
Number of stars is not update

### ISSUE THREAD

**Issue by yoanmarchal** [iankronquist/y-cloninator#16]
nice app but the number of stars for each repository is not updated

**Comment by iankronquist** [comment]
Good point. We just grab them the first time we populate the database. We'd have to hit the api endpoint each repo in the DB to get the updated number of stars if I'm not mistaken. It sounds like we need another periodic job. @tschuy thoughts?
